torch_geometric/datasets/email_eu_core.py

```diff
@@ -0,0 +1,104 @@
+import gzip
+import os
+
+import networkx as nx
+import torch
+from torch_geometric.data import InMemoryDataset, download_url
+from torch_geometric.utils.convert import from_networkx
+
+
+class Email_eu_core(InMemoryDataset):
+    """ This dataset is an e-mail communication network from a large European 
+    research institution, taken from the "Local Higher-order Graph Clustering." 
+    paper <https://www-cs.stanford.edu/~jure/pubs/mappr-kdd17.pdf>.  
+    Vertices indicate members of the institution and an edge between a pair 
+    of members indicates that they exchanged at least one email. 
+    Vertex labels indicate membership to one of the 42 departments.
+
+    Args:
+        root (string): Root directory where the dataset should be saved.
+        split (string): If :obj:`"train"`, loads the training dataset.
+            If :obj:`"val"`, loads the validation dataset.
+            If :obj:`"test"`, loads the test dataset. (default: :obj:`"train"`)
+        transform (callable, optional): A function/transform that takes in an
+            :obj:`torch_geometric.data.Data` object and returns a transformed
+            version. The data object will be transformed before every access.
+            (default: :obj:`None`)
+        pre_transform (callable, optional): A function/transform that takes in
+            an :obj:`torch_geometric.data.Data` object and returns a
+            transformed version. The data object will be transformed before
+            being saved to disk. (default: :obj:`None`)
+        pre_filter (callable, optional): A function that takes in an
```

Comment:
As the dataset only has a single example, I suggest to remove `pre_filter` as well.